docs/lightnode.md

```diff
@@ -0,0 +1,41 @@
+## Light-node mode
+
+The node offers an option for the user to start up the node in what's called 'light-node' mode. It is intended for those wishing to only use the wallet functionality, and disables certain components, and allows the node to ignore most of the network messages, drastically decreasing bandwidth consumption. Below follows a more in-depth explanation of the effects 'light-node' mode has on the process.
```

Comment:
Is light-mode intended only for wallet nodes?  Could dusk-explorer node benefit from it too?

Reply:
Dusk-explorer might be able to benefit from it, though it limits access to certain information (no mempool, no consensus messaging).